i2c/mcp9808_i2c/README.adoc

```diff
@@ -0,0 +1,37 @@
+= Attaching a MCP9808 digital temperature sensor via I2C
+
+This example code shows how to interface the Raspberry Pi Pico to the MCP9808 digital temperature sensor board. 
+======
+This example reads the ambient temperature value each second from the sensor and sets upper, lower and critical limits for the temperature and checks if alerts need to be raised. The CONFIG register can also be used to check for an alert if the critical temperature is surpassed.
+======
+
+== Wiring information
+
+Wiring up the device requires 7 jumpers, to connect VDD, GND, SDA, SCL, A0, A1 and A2 pins. An additional jumper could also be required to connect the Pico GND to the power rail on the breadboard. The example here uses I2C port 0, which is assigned to GPIO 4 (SDA) and 5 (SCL) in software. Power is supplied from the VSYS pin.
+
+
+
+[[mcp9808_i2c_wiring]]
+[pdfwidth=75%]
+.Wiring Diagram for MCP9808.
+image::mcp9808_i2c_bb.png[]
+
+== List of Files
+
+CMakeLists.txt:: CMake file to incorporate the example in to the examples build tree.
+mcp9808_i2c.c:: The example code.
+
+== Bill of Materials
+
+.A list of materials required for the example
+[[mcp9808-bom-table]]
+[cols=3]
+|===
+| *Item* | *Quantity* | Details
+| Breadboard | 1 | generic part
+| Raspberry Pi Pico | 1 | http://raspberrypi.org/
+| MCP9808 board| 1 | generic part

```

Comment:
You've used the Adafruit part, https://www.adafruit.com/product/1782, you should link to it.